pegasus/routes/form_routes.rb

```diff
@@ -16,7 +16,13 @@
     content_type :json
     cache_control :private, :must_revalidate, max_age: 0
     form = insert_or_upsert_form(kind, params)
-    JSON.load(form[:data]).merge(secret: form[:secret]).to_json
+    data = JSON.load(form[:data]).merge(secret: form[:secret])
+    if form[:kind] == "VolunteerContact2015"
```

Comment:
Are we re-using the same form template from 2015? Anything about the year that needs updating?

Reply:
Yep. Bad naming on my part. I had created this when we created the HocSignup form. While we bumped the HocSignup every year, we didn't bump the volunteer one.